ouroboros-consensus/src/Ouroboros/Storage/VolatileDB/Impl.hs

```diff
@@ -731,6 +742,6 @@ reverseMap file revMap mp = foldM go revMap (Map.toList mp)
        -> Either VolatileDBError (ReverseIndex blockId)
     go rv (offset, (size, BlockInfo {..})) = case Map.lookup bbid rv of
       Nothing -> Right $ Map.insert
-        bbid (InternalBlockInfo file offset size bslot bpreBid) rv
+        bbid (InternalBlockInfo file offset size bslot bpreBid bheaderOffset bheaderSize) rv
```

Comment:
Consider moving the construction of `InternalBlockInfo` to a `where` block and using record syntax.